src/RoadRegistry.Wms.ProjectionHost/appsettings.development.json

```diff
@@ -20,14 +20,18 @@
     {
       "Name": "Seq",
       "Args": {
-        "serverUrl": "http://seq:5341"
+        "serverUrl": "http://localhost:5341"
```

Comment:
`seq` only works inside the docker network - this file is normally for debugging purposes.